### PR TITLE
Fix 404 links on Command Cookbook, also fixes #5

### LINKED ISSUE #5: Incorrect links in Commands Cookbook page
Several links are incorrect now as we have moved to make site.

Like, current link for `documentation standards` is `https://make.wordpress.org/docs/documentation-standards/`

Correct link `https://make.wordpress.org/cli/handbook/documentation-standards/`

### PR DESCRIPTION
**Pull request by JayWood**
For now I've linked all 404 pages back to the original .org site, this is at least until a more robust structure is put in place for handling sub-directories like was on the old site.

### CODE REVIEW

File: commands-cookbook.md [hunk 1]
```diff
@@ -493,7 +493,7 @@ if ( defined( 'WP_CLI' ) && WP_CLI ) {
 	
 ### Add to the package index
 
-Standalone WP-CLI commands can be added to and installed from the [package index](/package-index/). The only technical requirement for being listed in the package index is to include a valid composer.json file with an autoload declaration. We recommended including `"type": "wp-cli-package"` to distinguish your project explicitly as a WP-CLI package.
+Standalone WP-CLI commands can be added to and installed from the [package index](http://wp-cli.org/docs/internal-api/wp-cli-add-hook//package-index/). The only technical requirement for being listed in the package index is to include a valid composer.json file with an autoload declaration. We recommended including `"type": "wp-cli-package"` to distinguish your project explicitly as a WP-CLI package.
```
Comment (danielbachhuber) [inline commands-cookbook.md]: This link should be `http://wp-cli.org/package-index/

File: commands-cookbook.md [hunk 2]
```diff
@@ -424,13 +424,13 @@ WP_CLI::add_command( 'find-unused-themes', $find_unused_themes_command, array(
...
+* `WP_CLI::add_command()` ([doc](http://wp-cli.org/docs/internal-api/wp-cli-add-hook//docs/internal-api/wp-cli-add-command/)) is used to register a `find-unused-themes` command to the `$find_unused_themes_command` closure. The `before_invoke` argument makes it possible to verify the command is running on a multisite install, and error if not.
+* `WP_CLI::error()` ([doc](http://wp-cli.org/docs/internal-api/wp-cli-add-hook//docs/internal-api/wp-cli-error/)) renders a nicely formatted error message and exits.
+* `WP_CLI::launch_self()` ([doc](http://wp-cli.org/docs/internal-api/wp-cli-add-hook//docs/internal-api/wp-cli-launch-self/)) initially spawns a process to get a list of all sites, then is later used to get the list of themes for a given site.
+* `WP_CLI::log()` ([doc](http://wp-cli.org/docs/internal-api/wp-cli-add-hook//docs/internal-api/wp-cli-log/)) renders informational output to the end user.
+* `WP_CLI\Utils\format_items()` ([doc](http://wp-cli.org/docs/internal-api/wp-cli-add-hook//docs/internal-api/wp-cli-utils-format-items/)) renders the list of unused themes after the command has completed its discovery.
```
Comment (danielbachhuber) [inline commands-cookbook.md]: The five links above aren't quite right.

File: commands-cookbook.md [hunk 3]
```diff
@@ -15,11 +15,11 @@ Creating your own custom WP-CLI command can be easier than it looks — and you
 
 ## Overview
 
-WP-CLI's goal is to offer a complete alternative to the WordPress admin; for any action you might want to perform in the WordPress admin, there should be an equivalent WP-CLI command. A **command** is an atomic unit of WP-CLI functionality. `wp plugin install` ([doc](/commands/plugin/install/)) is one such command, as is `wp plugin activate` ([doc](/commands/plugin/activate/)). Commands are useful to WordPress users because they can offer simple, precise interfaces for performing complex tasks.
+WP-CLI's goal is to offer a complete alternative to the WordPress admin; for any action you might want to perform in the WordPress admin, there should be an equivalent WP-CLI command. A **command** is an atomic unit of WP-CLI functionality. `wp plugin install` ([doc](/commands/plugin/install/)) is one such command, as is `wp plugin activate` ([doc](https://wp-cli.org/commands/plugin/activate/)). Commands are useful to WordPress users because they can offer simple, precise interfaces for performing complex tasks.
```
Comment (danielbachhuber) [inline commands-cookbook.md]: `wp plugin install` needs to be updated too

**Comment by danielbachhuber** [comment]
Thanks! Few issues to fix up

**Comment by JayWood** [comment]
@danielbachhuber you ninja, reviewed it while I was typing 😄 

I was going to ask, I seen in the other docs repo ( for the original .org ) that the content was organized into folders. Is that going to be the case for the new .org site as well? I'm unfamiliar with how the doc site generates it's information.

**Comment by danielbachhuber** [comment]
> I was going to ask, I seen in the other docs repo ( for the original .org ) that the content was organized into folders. Is that going to be the case for the new .org site as well?

Nope. 

>  I'm unfamiliar with how the doc site generates it's information.

Each Handbook page has a "Markdown Source" field:

image

The make/cli WordPress install is set up to import those every 15 minutes, so that's how the content here gets synced into WP.

**Comment by JayWood** [comment]
So I'm correct in assuming that on the WP side of things, you would need to physically create the sub-pages on the WordPress side, they're not created simply by creating a folder on the github repo?

**Comment by danielbachhuber** [comment]
> So I'm correct in assuming that on the WP side of things, you would need to physically create the sub-pages on the WordPress side, they're not created simply by creating a folder on the github repo?

Correct (for now at least). I went with the easiest possible implementation to start.

**Comment by JayWood** [comment]
@danielbachhuber if I've missed something let me know - but I think you're just slating this for the next release?

**Comment by danielbachhuber** [comment]
> if I've missed something let me know

Nah, forgot about it. Thanks for the reminder!

> but I think you're just slating this for the next release?

I use milestones to keep track of who contributed in which releases. The changes will go live right away.

**Comment by JayWood** [comment]
@danielbachhuber gotcha, thanks for the info

**Pull request merged**